### ISSUE TITLE
[DOC] initialize() is called in constructor not componentDidMount

### ISSUE THREAD

**Issue by ericls** [draft-js-plugins/draft-js-plugins#1102]
It says in the [doc](https://github.com/draft-js-plugins/draft-js-plugins/blob/master/HOW_TO_CREATE_A_PLUGIN.md#initialize ) that initialize allows users to initialize plugin once the editor is mounted. But looks like initialize is called before editor is mounted. This leads to problems such as `setReadOnly` being not usable when initializing a plugin.

There are work arounds to it, but it would be nice to have it clarified in the doc.

**Comment by juliankrispel** [comment]
Hmmmm, might actually be a good idea to change this so that we can use `setReadOnly`.

**Comment by juliankrispel** [comment]
I'm working on 3.0 of the editor so if this is a breaking change it'd be good to go in there...

**Comment by w-** [comment]
I would like to +1 this. I just got bitten by this because I'm doing SSR for a plugin which needs to dynamically load a JS file but client side.

https://github.com/zeit/next.js/issues/2473

the language in the doc makes me beleive it will be called after `componentDidMount`

is there another way to call stuff only after `componentDidMount` ?

@ericls what workarounds did you find for this?
